pkg/csi/service/wcp/controller.go

```diff
@@ -123,6 +123,17 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 		log.Errorf("checkAPI failed for vcenter API version: %s, err=%v", vc.Client.ServiceContent.About.ApiVersion, err)
 		return err
 	}
+
+	if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.TKGsHA) {
+		err = common.CheckAPI(vc.Client.ServiceContent.About.ApiVersion, common.TKGsHASupportedVCenterMajor,
```

Comment:
Have you verified CheckAPI call with version `8.0.0.0`

Reply:
let's remove this check, as the supervisor is bundled within vSphere release we do not need this version check.

Reply:
will remove the check of vSphere version per our offline discussion